src/autoscaler/scalingengine/server/scaling_handler_test.go

```diff
@@ -427,4 +429,55 @@ var _ = Describe("ScalingHandler", func() {
 			})
 		})
 	})
+
+	Describe("GetActiveSchedule", func() {
+		JustBeforeEach(func() {
+			handler.GetActiveSchedule(resp, req, map[string]string{"appid": "an-app-id"})
+		})
+
+		Context("when query database succeeds", func() {
+			BeforeEach(func() {
+				req, err = http.NewRequest(http.MethodGet, testUrlAppActiveSchedule, nil)
+				Expect(err).ToNot(HaveOccurred())
+
+				activeSchedule = &models.ActiveSchedule{
+					ScheduleId:         "a-schedule-id",
+					InstanceMin:        1,
+					InstanceMax:        5,
+					InstanceMinInitial: 3,
+				}
+
+				scalingEngineDB.GetActiveScheduleReturns(activeSchedule, nil)
+			})
+
+			It("returns 200 with active schedule in message body", func() {
+				Expect(resp.Code).To(Equal(http.StatusOK))
+
+				actualActiveSchedule := &models.ActiveSchedule{}
+				err = json.Unmarshal(resp.Body.Bytes(), actualActiveSchedule)
+
+				Expect(err).ToNot(HaveOccurred())
+				Expect(*actualActiveSchedule).To(Equal(*activeSchedule))
```

Comment:
You mean , we can save value in one of them,actualActiveSchedule := models.ActiveSchedule{} instead of actualActiveSchedule := &models.ActiveSchedule{}